_site/index.html

```diff
@@ -0,0 +1,205 @@
+<p>If you've got the <a href="https://www.rust-lang.org/tools/install">Rust compiler
+installed</a> then you can take some Rust
+source code:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#b48ead;">fn </span><span style="color:#8fa1b3;">main</span><span style="color:#c0c5ce;">() {
+</span><span style="color:#c0c5ce;">    println!(&quot;</span><span style="color:#a3be8c;">Hello, world!</span><span style="color:#c0c5ce;">&quot;);
+</span><span style="color:#c0c5ce;">}
+</span></pre>
+<p>and compile/run it with:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustup target add wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustc hello.rs</span><span style="color:#bf616a;"> --target</span><span style="color:#c0c5ce;"> wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> wasmtime hello.wasm
+</span><span style="color:#8fa1b3;">Hello,</span><span style="color:#c0c5ce;"> world!
+</span></pre>
+</div>
+</section>
+<section>
+<div class="container w-container">
+<h2>Features</h2>
+<ul>
+<li>
+<p><strong>Fast</strong>. Wasmtime is built on the optimizing <a href="https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/README.md">Cranelift</a> code generator to
+quickly generate high-quality machine code either at runtime or
+ahead-of-time. Wasmtime's runtime is also optimized for cases such as
+efficient instantiation, low-overhead transitions between the embedder and
+wasm, and scalability of concurrent instances.</p>
+</li>
+<li>
+<p><strong><a href="https://docs.wasmtime.dev/security.html">Secure</a></strong>. Wasmtime's development is strongly focused on the correctness of
+its implementation with 24/7 fuzzing donated by <a href="https://google.github.io/oss-fuzz/">Google's OSS Fuzz</a>,
+leveraging Rust's API and runtime safety guarantees, careful design of
+features and APIs through an <a href="https://github.com/bytecodealliance/rfcs">RFC process</a>, a <a href="https://bytecodealliance.org/security">security policy</a> in place
+for when things go wrong, and a <a href="https://docs.wasmtime.dev/stability-release.html">release policy</a> for patching older versions
+as well. We follow best practices for defense-in-depth and known
+protections and mitigations for issues like Spectre. Finally, we're working
+to push the state-of-the-art by collaborating with academic
+researchers to formally verify critical parts of Wasmtime and Cranelift.</p>
+</li>
+<li>
+<p><strong><a href="https://docs.rs/wasmtime/latest/wasmtime/struct.Config.html">Configurable</a></strong>. Wastime supports a rich set of APIs and build time
+configuration to provide many options such as further means of restricting
+WebAssembly beyond its basic guarantees such as its CPU and Memory
+consumption. Wasmtime also runs in tiny environments all the way up to massive
+servers with many concurrent instances.</p>
```

Comment:
The two "such as"s in the first sentence makes it a bit of a journey. Maybe something like:

Wasmtime uses sensible defaults, but can also be configured to provide more fine-grained control over things like CPU and memory consumption. Whether you want to run Wasmtime in a tiny environment or on massive servers with many concurrent instances, we've got you covered.